SCClassLibrary/Common/GUI/PlusGUI/Control/ServerPlusGUI.sc

```diff
@@ -1,17 +1,165 @@
 ServerTreeView {
 	var <window;
 	*new { |aServer, bounds=(Rect(128, 64, 400, 400)), interval=0.5|
-		var onClose, window;
+		var aServerTreeView, window;
 		window = Window.new(name.asString, bounds, scroll:true).front;
-		onClose = aServer.plotTreeView(interval, window.view, { defer {window.close}; });
-		window.onClose = {
-			onClose.value;
-		};
+		aServerTreeView = ServerTreeView.plotTreeView(aServer, interval, window, { defer { window.close } });
```

Comment:
I'd suggest the following interface:

```supercollider
*new { |aServer, bounds=(Rect(128, 64, 400, 400)), parent|
      ^super.newCopyArgs(aServer).makeWindow(bounds, parent)
}

makeWindow { |bounds, parent|
   ... here all the window building goes ...
}

start { |interval=0.5, actionIfFail|
 .... here all the actions go, like resp building ....
}

stop {
 .... actions ar ended ...
like:
			updater.stop;
			CmdPeriod.remove(updateFunc);
			resp.free;
}
```

Reply:
To understand your suggestion:
Which of the following use cases do you have in mind?
- Use case 1
   ```
   x = ServerTreeView(s)
   x.start
   x.stop 
   // x.close // <- not provided.
   ```
- Use case 2
   ```
   x = ServerTreeView(s)
   x.start
   x.stop
   x.close
   ```
- Use case 3
   ```
   x = ServerTreeView(s)
   x.close // x.start and x.stop is automatically done
   ```

I ask this question because there is no `interval` argument in the `*new` method in your suggestion:
>*new { |aServer, bounds=(Rect(128, 64, 400, 400)), parent|

Reply:
I would say that it is (2), and `close` will do the `stop`.
```
x = ServerTreeView(s).start;
x.close
```

So in principle one could make a user interface with a ServerTreeView without a booted server.

Reply:
It is a great idea!

Here is my implementation: https://github.com/supercollider/supercollider/pull/6294/commits/0325b39df812cfd73dac3e87738ffc67e16bf335

and here a demo video of test: https://www.dropbox.com/scl/fi/1cns3wlno0j53gg74t2cq/ServerTreeView-.plotTree-.plotTreeView.mov?rlkey=ceii91a8xnae6vwnv24ppxjkt&dl=0

Reply:
I have changed the behaviour of `s.plotTree` when it is evaluated while there is already a `s.plotTree`: https://github.com/supercollider/supercollider/pull/6294/commits/6193918ff634205a18a23ed69b95fc7b327f102e